os/arch/arm/src/armv8-m/up_releasepending.c

```diff
@@ -66,6 +66,12 @@
 #include "mpu.h"
 #endif
 
+#ifdef CONFIG_ARMV8M_TRUSTZONE
+#ifdef CONFIG_ARCH_CHIP_AMEBAD
```

Comment:
All the modified codes are enclosed with CONFIG_ARMV8M_TRUSTZONE, but this header requires CONFIG_ARCH_CHIP_AMEBAD. Which is correct?

Reply:
done